token-account-based/README.md

```diff
@@ -0,0 +1,200 @@
+# Account-based token scenario
+
+The account-based token smart contract demonstrates how to create and transfer fungible tokens using an account-based model. In an account-based model, there is an account for each participant that holds a balance of tokens.
+A mint transaction creates tokens in an account, while a transfer transaction debits the caller's account and credits another account.
+
+In this sample it is assumed that only one organization (played by Org1) is in a central banker role and can mint new tokens into their account, while any organization can transfer tokens from their account to a recipient's account.
+Accounts could be setup at the organization level or client identity level. In this sample accounts are setup at the client identity level, where every authorized client with an enrollment certificate from their organization has an account keyed by their client ID.
+The client ID is simply a base64-encoded concatenation of the issuer distinguished name and subject distinguished name from the client identity's enrollment certificate. The client ID can therefore be considered the payment address of a recipient.
```

Comment:
It might be worth it to try to explain distinguished name and subject in more plain language.

Reply:
in the private data tutorial, we refer to the common name and issuer of the identity certificate.

Reply:
Technically it is the distinguished name that is used, not the common name, but I agree we don't have to get technical here, removed 'distinguished'.